src/vos/vos_ts.h

```diff
@@ -335,7 +351,7 @@ vos_ts_set_get_entry(struct vos_ts_set *ts_set)
 {
 	struct vos_ts_set_entry	*entry;
 
-	if (ts_set == NULL || ts_set->ts_init_count == 0)
+	if (!vos_ts_in_tx(ts_set))
```

Comment:
Why drop the check "ts_set->ts_init_count == 0"?

Reply:
Ah, that's a good catch